pydicom/multival.py

```diff
@@ -49,6 +49,13 @@ def number_string_type_constructor(x):
         for x in iterable:
             self._list.append(type_constructor(x))
 
+    # TODO: Workaround for #951, to be removed when Python 2 not supported
```

Comment:
No need for a TODO here - on removing Python2 support we will check all `compat.in_py2` occurences anyway.